tests/test_compare_taxonomy.py

```diff
@@ -22,6 +22,7 @@ def test_basic(location):
     args.hit_list = hitlist
     args.genome_summary = summary_csv
     args.lineages_csv = utils.relative_file("tests/test-data/test-match-lineages.csv")
+    args.databases = [utils.relative_file('tests/test-data/loomba/LoombaR_2017__SID1050_bax__bin.11.fa.gz.matches.zip')]
```

Comment:
ok so the full db, which would usually be gtdb, becomes the matches.zip file here, which is sort of just a convenient test data stand in?

Reply:
exactly right - this didn't actually use the full db in the first place, this stage used matches output by a previous stage (which I was providing in a standard file location). But now it needs the original database so that the matching signatures can be picklisted out using the intermediate output from prefetch.